lib/dlss/capistrano/tasks/control_master.rake

```diff
@@ -17,6 +17,17 @@ namespace :deploy do
   end
 end
 
+namespace :shared_configs do
+  before :check, :setup_connection do
+    invoke 'controlmaster:setup'
```

Comment:
When I used the (read-only) `shared_configs:check` task against sul-embed prod (which is load balanced) with this line commented out, and no active SSH control master, I got two Duo prompts, instead of just one (because I was prompted by cap's attempt to connect to each server in parallel; whereas this `invoke` causes a shell out to `ssh` that sets up the controlmaster before capistrano has a chance to try connecting to each server as part of its default flow).